manifests/t/Torchsoft/RegistryWorkshop/5.0.1/Torchsoft.RegistryWorkshop.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Torchsoft.RegistryWorkshop
+PackageVersion: 5.0.1
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: http://www.torchsoft.com/download/RegistryWorkshop.exe
+  InstallerSha256: CABBB998557C5E883200D082853CA0DB1A9F0DB66332A0731019B7485C2E95D0

```

Comment:
```suggestion
  InstallerSha256: CABBB998557C5E883200D082853CA0DB1A9F0DB66332A0731019B7485C2E95D0
  ProductCode: Registry Workshop
```

Reply:
Where should I add it to? I don't see any blank spot in 3 files.

Reply:
If you are the Developer of this software your installer needs to create a registry string at `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Registry Workshop` called DisplayVersion and Publisher

![VirtualBoxVM_RJOSqtq7Tc](https://user-images.githubusercontent.com/15158490/162747673-003b46e2-655d-4995-8750-5bcbe51159e1.png)

If you are just a user you need to contact the Developer and ask them to fix the issue. Additionally because the software does not set the DisplayVersion it will cause a upgrade loop.

Reply:
Just contacted the developers. If the issue is fixed, I'll make another commit.